Products/ZenUI3/browser/resources/js/zenoss/eventclasses/ClassPanels.js

```diff
@@ -825,6 +826,41 @@ Ext.onReady(function(){
                         cls: 'transform-headers',
                         text: 'Current Transform:'
                     });
+                    me.add({
+                        xtype: 'button',
+                        disabled: true,
+                        hidden: Zenoss.Security.doesNotHavePermission('Manage DMD'),
+                        text: _t('Change Transform state'),
+                        listeners : {
+                            click: function(el) {
+                                Zenoss.remote.EventClassesRouter.changeTransformState({'uid':uid}, function(response){
+                                    if(response.success){
+                                        if(response.data === true){
+                                            Zenoss.message.info(_t('State changed. Transform is now enabled.'));
+                                            el.setText('Disable Transform');
+                                        }
+                                        else{
+                                            Zenoss.message.info(_t('State changed. Transform is now disabled.'));
+                                            el.setText('Enable Transform');
+                                        }
+                                    }
+                                });
+                            },
+                            afterrender: function(el) {
+                                Zenoss.remote.EventClassesRouter.getTransformState({'uid':uid}, function(response){
+                                    if(response.success){
+                                        if(response.data === true){
+                                            el.setText('Disable Transform');
+                                        }
+                                        else{
+                                            el.setText('Enable Transform');
+                                        }
+                                        el.enable()
+                                    }
+                                });
+                            }
+                        }
+                    });
```

Comment:
This code suggests that the button is always visible.  It has options and text for enabling and disabling event transforms.